app/python/synchrona/main.py

```diff
@@ -57,6 +58,11 @@
     allow_headers=["*"],
 )
 
+# start process for handling Status LEDs color on synchrona board
+os.system("ps axf | grep \"ad-synchrona-14-leds\.sh\" | grep -v grep | awk '{print \"kill -9 \" $1}' | sh")
+subprocess.Popen(['bash', 'ad-synchrona-14-leds.sh'], stderr = open(os.devnull, 'w'),
+    stdout = open(os.devnull, 'w'))
+
```

Comment:
This looks very odd and hacky to me... Is there any reason why we want to kill the script and start it again every time we get in here?

The way I see it, this script looks like it's something that should be started/stopped by the synchrona systemd service,